Packs/Armorblox/Integrations/Armorblox/Armorblox.py

```diff
@@ -39,11 +38,12 @@ def get_incidents(self, orderBy='ASC', pageSize=None, pageToken=None, first_fetc
 
         if pageSize:
             request_params['pageSize'] = pageSize
-        response_json, _ = self.incidents.list_resource(PATH, params=request_params)
+
+        response_json, next_page_token, total_count = self.incidents.list(params=request_params)
         if response_json is None:
-            return []
+            return [], next_page_token
```

Comment:
If `response_json` is `None`, then why do we need to send `next_page_token` back? Also, `response_json` will be `[]` by default & `next_page_token` is `None` by default. So you can send them back directly - no need for this `if` check.